customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -427,6 +427,8 @@ public CustomerDao build() {
 
     @DynamoDbBean
     public static class DoiAgentDao implements DoiAgent {
+
+        private String url;
```

Comment:
I know the url to DataCite is repeated for all entries in the existing secret, but I wonder if the at any point will have different url's for a customer in the same environment. I would check this. It would be very cumbersome if the user has to type in this url in the UI every time they configure DOI using DataCite. No blocker, just try to find out, and we might fix this later if needed.